compiler/testSuite/Voptorder00.bal

```diff
@@ -0,0 +1,19 @@
+import ballerina/io;
+
+public function main() {
+    int? n1 = 1;
+    int? n2 = ();
+    io:println(n1 < n2); // @output false
```

Comment:
Should have the other order ops for case where one operand is ().

Should test  left operand being nil.

Should test the case where both are nil.

Reply:
Fixed